freqtrade/templates/sample_strategy.py

```diff
@@ -17,7 +17,8 @@
 class SampleStrategy(IStrategy):
     """
     This is a sample strategy to inspire you.
-    More information in https://github.com/freqtrade/freqtrade/blob/develop/docs/bot-optimization.md
+    More information in:
+    https://github.com/freqtrade/freqtrade/blob/develop/docs/strategy-customization.md
```

Comment:
Honestly, i'd rather use https://www.freqtrade.io/en/latest/strategy-customization/ ... 

That's the "rendered" version for this - which is ususally the better option

Reply:
Yes looks cleaner :+1: Link updated